app/controllers/api/account.php

```diff
@@ -292,8 +292,6 @@
         $appId = $project->getAttribute('usersOauth2'.\ucfirst($provider).'Appid', '');
         $appSecret = $project->getAttribute('usersOauth2'.\ucfirst($provider).'Secret', '{}');
 
-        $appSecret = \json_decode($appSecret, true);
```

Comment:
@eldadfux Is there any reason this line was added? It was breaking the oAuth flow in the Linkedin case while testing

Reply:
Maybe because of the Apple OAuth adapter? can you check if Facebook or other providers has the same issue?

Reply:
Yep, this line can be removed, not needed with the intro of DB filters.